pages/program/abstracts/tutorials.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: page
+title: Tutorials
+description: 
+menubar: program
+permalink: program/tutorials/
+menubar_toc: true
+set_last_modified: true
+---
+
```

Comment:
Can you please add something to the top of this page that says that tutorials are virtual / happening in the two weeks before the conference / will be free to attend?

Reply:
Sure. I used the copy from last year and removed the date.